client/dashboard/profile-wizard/steps/business-details.js

```diff
@@ -42,12 +44,13 @@ class BusinessDetails extends Component {
 
 	async onContinue( values ) {
 		const { createNotice, goToNextStep, isError, updateProfileItems } = this.props;
-		const { facebook, mailchimp, other_platform, product_count, selling_venues } = values;
+		const { facebook, mailchimp, other_platform, product_count, revenue, selling_venues } = values;
 		const businessExtensions = this.getBusinessExtensions( values );
 
 		recordEvent( 'storeprofiler_store_business_details_continue', {
 			product_number: product_count,
 			already_selling: 'no' !== selling_venues,
+			revenue,
```

Comment:
Should we include the currency here as well?  Either as a separate param or part of the revenue range?

Reply:
Fixed in https://github.com/woocommerce/woocommerce-admin/pull/2763/commits/19ae63deb6c634599d4b28df76c19cc67c876543.